docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.8"
+services:
+  extract_diffs:
+    image: makerdao/vdb-extract-diffs:latest # (needs wait-for-it.sh)
+    environment:
+      # Remote client would need to be statediff enabled
+      - CLIENT_IPCPATH=ws://geth:8546/
+      - STORAGEDIFFS_SOURCE=geth
+      - DATABASE_NAME=vdb
+      - DATABASE_HOSTNAME=db
+      - DATABASE_PORT=5432
+      - DATABASE_USER=vdb
+      - DATABASE_PASSWORD=vdb
+    depends_on:
+      - geth 
+    command: ["./wait-for-it.sh", "db:5432", "--strict", "--", "./startup_script.sh"]
+
+  geth:
+    build:
+      .
+    command: ["--http", "--http.addr", "0.0.0.0", "--ws", "--ws.addr", "0.0.0.0", "--syncmode", "full"]
+
+  db:
+    image: postgres:12.1-alpine
+    ports:
```

Comment:
I think you can omit this mapping if nothing on the host is connecting to this port.

Reply:
Cool yeah.. I added this back when i thought I was having a problem accessing the DB.